rpi-convert-stage-5.sh

```diff
@@ -66,8 +66,16 @@ build_uboot_files() {
 	fdt addr ${fdt_addr} && fdt get value bootargs /chosen bootargs
 	run mender_setup
 	mmc dev ${mender_uboot_dev}
-	load ${mender_uboot_root} ${kernel_addr_r} /boot/zImage
-	bootz ${kernel_addr_r} - ${fdt_addr}
+	if [ -f /boot/zImage ]; then
```

Comment:
This scrip is running in U-boot hush environment and I would not expect above to work at all. Have you verified this?

You probably need to do something like this,

```
if load ${mender_uboot_root} ${kernel_addr_r} /boot/zImage; then
    bootz ${kernel_addr_r} - ${fdt_addr}
elif load ${mender_uboot_root} ${kernel_addr_r} /boot/uImage; then
    bootm ${kernel_addr_r} - ${fdt_addr}
fi
```

Reply:
Probably you're right - the code commited was proposed by a collegue. I presumed he tested it but he may not. I'll look into it and come back to you

Reply:
You were right - the code was not tested - sorry for the inconvenience and thanks a lot for making sure the quality of the mender software is good!